includes/discovery/sensors/frequency/rs.inc.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+$oids = snmpwalk_cache_oid($device, 'cmdExcFrequency', [], 'RS-XX8000-DVB-TX-MIB');
```

Comment:
Any reason you couldn't do this in yaml?

Reply:
is $id a string?

Reply:
Yes, its in the description of the PR ;)

I had to use php for frequency due to index being a letter (a) translated to 1 in num oid.